grafana-plugin/src/style/utils.css

```diff
@@ -1,3 +1,7 @@
+.link {
+  text-decoration: none !important;
```

Comment:
plugin links don't have underlines, it comes from grafana styles 'a { text-decoration: none;}'

Reply:
hmm, but they do. It's a different style affecting it 
![image](https://user-images.githubusercontent.com/40542072/225061157-51f406ff-c288-4397-a706-73c41d0825e8.png)